.gitignore

```diff
@@ -8,3 +8,5 @@ util/n64/Yay0decompress
 *.n64
 *.yaml
 *.z64
+filesystem/
```

Comment:
these (`filesystem/`, `asm/`) shouldn't be ignored in the splat dir itself, I would think

Reply:
Ah, right. I removed those lines.